tools/k8s_collectors/k8s-splunk-collector-helper.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import print_function
 import os
 import sys, getopt
+import subprocess
```

Comment:
i have changed from os.popen to subprocess. shell is not set. am hoping this will fix some of the security issues. but this script will be run by customer when we collect logs and metrics. so I do not think this affects any way .